### PR TITLE
Fixing ReferenceError with Promise when running slap on Node 0.10

### PR DESCRIPTION
**Pull request by didibus**

### CODE REVIEW

File: lib/cli.js [hunk 1]
```diff
@@ -59,7 +59,16 @@ if (opts.perf.profile && process.execArgv.indexOf('--prof') === -1) {
   return;
 }
 
-var Promise = require('bluebird');
+try {
+  var Promise = require('bluebird');
+} catch (e) {
+  if (e instanceof ReferenceError) {
```
Comment (dbkaplun) [inline lib/cli.js]: Why would a `ReferenceError` occur, and how does this logic fix it?

Comment (didibus) [inline lib/cli.js]: I wrongly assumed the require was what was throwing the Reference error. Now I see it just sets the Promise and the Reference error is thrown later on. I changed it.

**Comment by dbkaplun** [comment]
Thanks @didibus!

**Comment by InfernoZeus** [comment]
This also fixes the same issue on Ubuntu 14.04 LTS - any chance of a new release tag for it @didibus?

**Comment by dbkaplun** [comment]
v0.1.60 has been released, thanks guys

**Comment by InfernoZeus** [comment]
Awesome, thanks!

**Pull request merged**